src/typings.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'styled-components' {
+  export type StylisPlugin = (context: number, content: string) => string | void;
+
+  export interface StyleSheetManagerProps {
+    children?: React.ReactNode;
+    disableCSSOMInjection?: boolean | undefined;
+    disableVendorPrefixes?: boolean | undefined;
+    stylisPlugins?: StylisPlugin[] | undefined;
+    sheet?: ServerStyleSheet | undefined;
+    target?: HTMLElement | ShadowRoot | undefined;
```

Comment:
Do they need undefined here if they are optional?

Reply:
I do not think that they need it. I will remove them.